F-Droid v2.10.5 crashes on start

# Bug report

## Describe your environment

* Device: Samsung A52s
* OS name and version: Android 13
* IVPN app version: 2.10.5
  
## Describe the problem

After the update to 2.10.5 the app immediately crashes on start. Clearing app data didn't help.

### Steps to reproduce:

1. Install 2.10.5
2. Open the app
  
### Observed Results:

I don't have access to my laptop right now but if it's needed I can send adb logs.
  
### Expected Results:

 The app should start 😄

Here are the logs from adb:

```
12-17 19:57:58.345 24409 24409 E AndroidRuntime: FATAL EXCEPTION: main
12-17 19:57:58.345 24409 24409 E AndroidRuntime: Process: net.ivpn.client, PID: 24409
12-17 19:57:58.345 24409 24409 E AndroidRuntime: java.lang.RuntimeException: Unable to get provider io.sentry.android.core.SentryInitProvider: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installProvider(ActivityThread.java:8333)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:7833)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7582)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2400)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8762)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.initConfigurations(Sentry.java:192)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.init(Sentry.java:162)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.init(Sentry.java:110)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryAndroid.init(SentryAndroid.java:59)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryAndroid.init(SentryAndroid.java:32)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryInitProvider.onCreate(SentryInitProvider.java:25)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2522)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2492)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryInitProvider.attachInfo(SentryInitProvider.java:42)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installProvider(ActivityThread.java:8328)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	... 11 more
12-17 19:57:58.346 24409 24435 I Sentry  : sentry-debug-meta.properties file was not found.
12-17 19:57:58.348 24409 24435 D Sentry  : SU isn't found on this Device.
12-17 19:57:58.371 24409 24409 I Process : Sending signal. PID: 24409 SIG: 9

```

Thanks for making internet better

here are the full logs if it helps more:

```
--------- beginning of main
12-17 19:57:58.281 24409 24409 I net.ivpn.client: Using CollectorTypeCC GC.
12-17 19:57:58.284 24409 24409 E net.ivpn.client: Not starting debugger since process cannot load the jdwp agent.
12-17 19:57:58.289 24409 24409 D ActivityThread: setConscryptValidator
12-17 19:57:58.289 24409 24409 D ActivityThread: setConscryptValidator - put
12-17 19:57:58.302 24409 24409 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10578; state: ENABLED
12-17 19:57:58.312 24409 24409 I net.ivpn.client: Failed to add image file: Failed to find OatDexFile for DexFile /data/app/~~fY9v7w_0OZ1U7Otsc74esg==/net.ivpn.client-XxI-5Vqj042lvEt39dzF9w==/base.apk (canonical path /data/app/~~fY9v7w_0OZ1U7Otsc74esg==/net.ivpn.client-XxI-5Vqj042lvEt39dzF9w==/base.apk) in OatFile /data/app/~~a689lZWL8btdMkMiMryFEw==/net.ivpn.client-TFX8lK9Ltg198NfLgXXhTw==/oat/arm64/base.odex
12-17 19:57:58.314 24409 24409 D nativeloader: Configuring clns-4 for other apk /data/app/~~a689lZWL8btdMkMiMryFEw==/net.ivpn.client-TFX8lK9Ltg198NfLgXXhTw==/base.apk. target_sdk_version=33, uses_libraries=, library_path=/data/app/~~a689lZWL8btdMkMiMryFEw==/net.ivpn.client-TFX8lK9Ltg198NfLgXXhTw==/lib/arm64:/data/app/~~a689lZWL8btdMkMiMryFEw==/net.ivpn.client-TFX8lK9Ltg198NfLgXXhTw==/base.apk!/lib/arm64-v8a, permitted_path=/data:/mnt/expand:/data/user/0/net.ivpn.client
12-17 19:57:58.326 24409 24409 V GraphicsEnvironment: ANGLE Developer option for 'net.ivpn.client' set to: 'default'
12-17 19:57:58.326 24409 24409 V GraphicsEnvironment: ANGLE GameManagerService for net.ivpn.client: false
12-17 19:57:58.326 24409 24409 V GraphicsEnvironment: Updatable production driver is not supported on the device.
12-17 19:57:58.329 24409 24409 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-17 19:57:58.330 24409 24409 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-17 19:57:58.331 24409 24409 I MultiDex: VM with version 2.1.0 has multidex support
12-17 19:57:58.331 24409 24409 I MultiDex: Installing application
12-17 19:57:58.331 24409 24409 I MultiDex: VM has multidex support, MultiDex support library is disabled.
12-17 19:57:58.332 24409 24409 I Sentry  : Retrieving auto-init from AndroidManifest.xml
12-17 19:57:58.345 24409 24409 D AndroidRuntime: Shutting down VM
--------- beginning of crash
12-17 19:57:58.345 24409 24409 E AndroidRuntime: FATAL EXCEPTION: main
12-17 19:57:58.345 24409 24409 E AndroidRuntime: Process: net.ivpn.client, PID: 24409
12-17 19:57:58.345 24409 24409 E AndroidRuntime: java.lang.RuntimeException: Unable to get provider io.sentry.android.core.SentryInitProvider: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installProvider(ActivityThread.java:8333)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:7833)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:7582)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2400)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8762)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.initConfigurations(Sentry.java:192)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.init(Sentry.java:162)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.Sentry.init(Sentry.java:110)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryAndroid.init(SentryAndroid.java:59)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryAndroid.init(SentryAndroid.java:32)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryInitProvider.onCreate(SentryInitProvider.java:25)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2522)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.content.ContentProvider.attachInfo(ContentProvider.java:2492)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at io.sentry.android.core.SentryInitProvider.attachInfo(SentryInitProvider.java:42)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	at android.app.ActivityThread.installProvider(ActivityThread.java:8328)
12-17 19:57:58.345 24409 24409 E AndroidRuntime: 	... 11 more
12-17 19:57:58.346 24409 24435 I Sentry  : sentry-debug-meta.properties file was not found.
12-17 19:57:58.348 24409 24435 D Sentry  : SU isn't found on this Device.
12-17 19:57:58.371 24409 24409 I Process : Sending signal. PID: 24409 SIG: 9

```

--- a/core/build.gradle
+++ b/core/build.gradle
@@ -30,8 +30,8 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 128
-        versionName "2.10.5"
+        versionCode 129
+        versionName "2.10.6"
         ndkVersion "25.1.8937393"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"

--- a/fdroid/build.gradle
+++ b/fdroid/build.gradle
@@ -31,8 +31,8 @@ android {
         applicationId "net.ivpn.client"
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 128
-        versionName "2.10.5"
+        versionCode 129
+        versionName "2.10.6"
         ndkVersion "25.1.8937393"
 
         manifestPlaceholders = [SENTRY_DSN: keystoreProperties['sentry.dsn']]

--- a/fdroid/src/main/AndroidManifest.xml
+++ b/fdroid/src/main/AndroidManifest.xml
@@ -20,6 +20,10 @@
         android:hasFragileUserData="false"
         tools:ignore="HardcodedDebugMode">
 
+        <meta-data
+            android:name="io.sentry.dsn"
+            android:value="${SENTRY_DSN}" />
+
     </application>
 
 </manifest>

--- a/liboqs-android/build.gradle
+++ b/liboqs-android/build.gradle
@@ -9,8 +9,8 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 128
-        versionName "2.10.5"
+        versionCode 129
+        versionName "2.10.6"
         ndkVersion "25.1.8937393"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"

--- a/site/build.gradle
+++ b/site/build.gradle
@@ -31,8 +31,8 @@ android {
         applicationId "net.ivpn.client"
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 128
-        versionName "2.10.5"
+        versionCode 129
+        versionName "2.10.6"
         ndkVersion "25.1.8937393"
 
         manifestPlaceholders = [SENTRY_DSN: keystoreProperties['sentry.dsn']]

--- a/store/build.gradle
+++ b/store/build.gradle
@@ -32,8 +32,8 @@ android {
         applicationId "net.ivpn.client"
         minSdkVersion 21
         targetSdkVersion 33
-        versionCode 128
-        versionName "2.10.5"
+        versionCode 129
+        versionName "2.10.6"
         ndkVersion "25.1.8937393"
 
         manifestPlaceholders = [SENTRY_DSN: keystoreProperties['sentry.dsn']]

